### ISSUE TITLE
Add profileFields to config for Facebook auth

### ISSUE THREAD

**Issue by ekryski** [feathersjs/generator-feathers#138]
See [the explanation in passport-facebook](https://github.com/jaredhanson/passport-facebook/#how-do-i-obtain-a-user-profile-with-specific-fields).

By default we probably want to set:

``` json
"profileFields": ["id", "first_name", "last_name", "name", "email", "gender", "birthday", "picture", "permissions"]
```

See [Facebook User docs](https://developers.facebook.com/docs/graph-api/reference/v2.5/user) for more detail.

**Comment by supasate** [comment]
Let me help on this issue.

**Comment by daffl** [comment]
This has been closed via #155 and will be released shortly. Thanks @supasate!
